.evergreen/pkgconfig-tests.sh

```diff
@@ -46,25 +46,33 @@ SRC_PATH="$(system_path ../)"
 $CMAKE -DBUILD_VERSION=1.18.0-pre -DENABLE_MONGOC=OFF -DCMAKE_BUILD_TYPE=RelWithDebInfo $ADDITIONAL_CMAKE_FLAGS -DCMAKE_INSTALL_PREFIX="$INSTALL_PATH" "$SRC_PATH"
 $CMAKE --build . --target install --config RelWithDebInfo
 
-# Build libmongocrypt, static linking against libbson
+# Build libmongocrypt, static linking against libbson and configured for the PPA
 cd $pkgconfig_tests_root/libmongocrypt-cmake-build
 PREFIX_PATH="$(system_path $pkgconfig_tests_root/install)"
 INSTALL_PATH="$(system_path $pkgconfig_tests_root/install/libmongocrypt)"
 SRC_PATH="$(system_path $libmongocrypt_root)"
-$CMAKE -DENABLE_SHARED_BSON=OFF -DCMAKE_BUILD_TYPE=RelWithDebInfo $ADDITIONAL_CMAKE_FLAGS -DCMAKE_PREFIX_PATH="$PREFIX_PATH" -DCMAKE_INSTALL_PREFIX="$INSTALL_PATH" "$SRC_PATH"
+$CMAKE -DENABLE_SHARED_BSON=OFF -DENABLE_BUILD_FOR_PPA=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo $ADDITIONAL_CMAKE_FLAGS -DCMAKE_PREFIX_PATH="$PREFIX_PATH" -DCMAKE_INSTALL_PREFIX="$INSTALL_PATH" "$SRC_PATH"
 $CMAKE --build . --target install --config RelWithDebInfo
+find ${PREFIX_PATH} -name libbson-static-1.0.a -execdir cp {} $(dirname $(find ${INSTALL_PATH} -name libmongocrypt-static.a )) \;
```

Comment:
IIUC this copies libbson-static-1.0.a into the install directory of libmongocrypt. Why is that necessary?

Reply:
Because we now install the libbson and libmongocrypt builds into separate directories (to make sure the packager script does not pick up extraneous components), a program building with the `libmongocrypt.pc` module would need the `libbson-static-1.0.a` library to be located there or the build will fail:

```
[2021/09/02 22:23:10.588] + gcc -DBSON_STATIC -DMONGOCRYPT_STATIC_DEFINE -DKMS_MSG_STATIC -I/data/mci/951753fb2fbc9d90761587fce32befe3/libmongocrypt/pkgconfig_tests/install/include/libbson-1.0 -I/data/mci/951753fb2fbc9d90761587fce32befe3/libmongocrypt/pkgconfig_tests/install/libmongocrypt/include/mongocrypt -o example-state-machine test/example-state-machine.c /data/mci/951753fb2fbc9d90761587fce32befe3/libmongocrypt/pkgconfig_tests/install/libmongocrypt/lib64/libmongocrypt-static.a /data/mci/951753fb2fbc9d90761587fce32befe3/libmongocrypt/pkgconfig_tests/install/libmongocrypt/lib64/libkms_message-static.a /data/mci/951753fb2fbc9d90761587fce32befe3/libmongocrypt/pkgconfig_tests/install/libmongocrypt/lib64/libbson-static-1.0.a /usr/lib64/librt.so -pthread -lpthread -lssl -lcrypto
[2021/09/02 22:23:10.588] gcc: error: /data/mci/951753fb2fbc9d90761587fce32befe3/libmongocrypt/pkgconfig_tests/install/libmongocrypt/lib64/libbson-static-1.0.a: No such file or directory
```

Depending on what we determine in terms of whether including `libbson-static-1.0.a` in the libmongocrypt PPA distribution and then requiring consumers to link directly to `libbson-static-1.0.a` constitutes overlinking, this may become unnecessary at some future point.